content/guides/docker-compose.mdx

```diff
@@ -95,3 +95,46 @@ When using `docker compose build` with Depot, there are a few things to be aware
 3. It's not possible to use multiple different Depot projects for different Compose services with `docker compose build`.
 
 However, `depot configure-docker` does directly integrate with any tools that use Docker Buildx, so if you are unable to use `depot bake --load` or otherwise need full Buildx compatibility with other tools, this is a good option.
+
+## Building and testing `docker compose` on GitHub Actions
+
+As we described above, the `depot bake` command can accept a `docker-compose.yml` file and build all services in parallel. However, by default this command will not load the images back into the local Docker daemon, which is necessary for running `docker compose` commands.
```

Comment:
So it is possible to `depot bake -f ./docker-compose.yml --load` which will load the images back to the local Docker daemon, which may be more efficient than `--save` + `depot pull` (I think this is mentioned above), _however_ that breaks if you want to build the image in one job and then use it in a second job within the workflow, like maybe you have a matrix job and don't want to trigger a large build on each.

It may be worth updating this to something like:

```yaml
name: Depot example compose
on: push

permissions:
  contents: read
  id-token: write
  packages: write

jobs:
  build-services:
    runs-on: ubuntu-22.04
    outputs:
      build-id: ${{ steps.bake.outputs.build-id }}
    steps:
      - uses: actions/checkout@v3
      - uses: depot/setup-action@v1
      - name: Build, cache, and save all compose images to the depot ephemeral registry.
        uses: depot/bake-action@v1
        id: bake
        with:
          files: docker-compose.yml
          save: true

  test:
    runs-on: depot-ubuntu-22.04
    needs: [build-services]
    steps:
      - uses: actions/checkout@v3
      - uses: depot/setup-action@v1
      - name: Pull all compose service images locally from the ephemeral registry.
        uses: depot/pull-action@v1
        with:
          build-id: ${{ needs.build-services.outputs.build-id }}
      - name: Run compose up (images should not rebuild)
        run: |
          docker compose up
      - name: If successful push the srv1 compose service target image to ghcr.io from ephemeral registry
        run: |
          echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $ --password-stdin
          depot push --target srv1 -t ghcr.io/depot/srv1:latest ${{ steps.build.outputs.build-id }}
```

Reply:
Excellent point/example!